styles/index.css

```diff
@@ -12,7 +12,181 @@ body {
   background-color: #e5e5e5;
 }
 
-/* header */
+/* main */
+#main {
+  display: flex;
+  flex-direction: column;
+}
+
+/* CLASS TAKE PRECEDE */
+.nav-buttons {
+  margin: 12px 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-buttons li {
+  height: 40px;
+  width: auto;
+  border: 1px solid #8993a4;
```

Comment:
- [ ] Kindly remove this border as it is not in the figma template. You are expected to stick to design template as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/235304355-2bb3f482-d501-4eea-9c9b-a5f62d6b08de.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/235304371-b2e30eee-a366-45df-9bff-d84d5688e783.png)